src/vivarium_csu_ltbi/tools/number_and_percent_under5_with_hh_AcTB.py

```diff
@@ -0,0 +1,70 @@
+import pandas as pd, numpy as np
+from db_queries import get_population
+
+def mergeCountry(df1, df2):
+    if "random_seed" in df1.columns:
+        del df1['random_seed']
+    if "random_seed" in df2.columns:
+        del df2['random_seed']
+    df = (df1.reset_index().set_index(['input_draw_number', 'random_seed', 'ltbi_treatment_scale_up.scenario'])
+              .add(
+              df2.reset_index().set_index(['input_draw_number', 'random_seed', 'ltbi_treatment_scale_up.scenario']),
+                  fill_value=0,
+          ))
+    assert not np.any(df.isnull())
+    return df.reset_index()
+
+def clean(data):
+    del data['random_seed']
+    new_data = data.reset_index().groupby(['input_draw_number', 'ltbi_treatment_scale_up.scenario']).sum()
+    new_data.drop(list(new_data.filter(like = 'person_time')), axis = 1, inplace = True)
+    return new_data
+
+def process(country, df, data, location_dict):
+    df_data = df.filter(like = "hhtb", axis = 1)
+    new_data = df_data.stack().reset_index().rename(columns = {0:'value', 'level_2' : 'information'})
+    new_data['age_group'] = new_data["information"].str.extract('^.*age_group_([a-z]*_?[a-z]*[0-9]*_?t?o?_?[0-9]*)', expand=True)
+    new_data['hh_status'] = new_data["information"].str.extract("^.*_([a-z]*)_to_hhtb")
+    #calculate proportion
+    num_u5 = new_data[new_data['age_group'].isin(['early_neonatal_', 'late_neonatal_', 'post_neonatal_', '1_to_4'])]["value"].sum()
+    df_u5= new_data[new_data['age_group'].isin(['early_neonatal_', 'late_neonatal_', 'post_neonatal_', '1_to_4'])].groupby("hh_status").sum().reset_index()
+    num_u5_hhtb = df_u5.loc[df_u5["hh_status"] == "exposed","value"][0]
+    hhtb_prop = num_u5_hhtb/num_u5
+    #rescale to national level
+    pop = get_population(location_id= location_dict.get(country),age_group_id=1,sex_id=3,gbd_round_id=5,

```

Comment:
It seems like you were using GBD 2017 both sexes, under-5 national population, i wonder is that match to the proportion you calculated? To be more specific, which scenario, sex, and year are you picking from sim person-time measure to calculate the proportion? For rescaling of `u5_hhtb` to national level population, i used 2019 sim baseline results with sex stratification.

Reply:
I used all year info and counts measure.